src/_includes/web-setup-deprecate.md

```diff
@@ -1,2 +1,2 @@
 {:.bs-callout-warning}
-The Web Setup Wizard is deprecated as of Magento 2.3.6 and removed in Magento 2.4.0. As an alternative, use the command line to [install](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli.html) or [upgrade](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html) Magento, as well as [extensions](https://devdocs.magento.com/extensions/install/).
+The Web Setup Wizard was deprecated in Magento 2.3.6 and is removed in Magento 2.3.7. After it is removed, you must use the command line to [install](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli.html) or [upgrade](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html) Magento, as well as [modules](https://devdocs.magento.com/guides/v2.3/comp-mgr/module-man/compman-checklist.html) and [extensions](https://devdocs.magento.com/guides/v2.3/comp-mgr/extens-man/extensman-checklist.html).
```

Comment:
I'm not sure if it's actually being removed or not. I would revert changes to this message to be safe or confirm with @chernenm.

Reply:
This is the text that will be displayed in the Admin according to the Dev ticket (MC-40598) ` “Please note Magento Web Setup Wizard has been removed since version 2.3.7.”.`

Reply:
@chernenm  Can you please verify this messaging for the User Guide for 2.3.7?

Reply:
You're right @leslietilling. I totally missed that part of the description.